evap/results/tests/test_views.py

```diff
@@ -264,6 +265,38 @@ def make_course_with_evaluations(unique_suffix):
         caches["sessions"].clear()
         caches["results"].clear()
 
+    def test_evaluation_weight_sums(self):
+        """
+        ensures that the sum of evaluation weights add up to 100%,
+        even if some evaluations are not shown to the user
+        """
+        student = baker.make(UserProfile, email="student@institution.example.com")
+        course = baker.make(Course)
+        evaluations = [
+            (
+                baker.make(
+                    Evaluation,
+                    course=course,
+                    name_en=f"evaluation{i}",
+                    name_de=f"evaluation{i}",
+                    state=Evaluation.State.PUBLISHED,
+                    weight=5 - i,  # weights 1 to 5
+                    is_single_result=True,
+                )
+            )
+            for i in range(0, 5)
+        ]
+        for evaluation in evaluations:
+            baker.make(RatingAnswerCounter, contribution=evaluation.general_contribution, answer=2, count=2)
```

Comment:
The model bakery library has a feature to create multiple instances at once, so you don't have to write the loops yourself:
```suggestion
        names = [f"evaluation{i}" for i in range(5)]
        evaluations = baker.make(
            Evaluation,
            course=course,
            name_en=iter(names),
            name_de=iter(names),
            state=Evaluation.State.PUBLISHED,
            weight=iter(range(5, 0, -1)),
            is_single_result=True,
            _quantity=5,
        )
        contributions = [e.general_contribution for e in evaluations]
        baker.make(RatingAnswerCounter, contribution=iter(contributions), answer=2, count=2, _quantity=len(evaluations))

```